pkgs/development/python-modules/geojson/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "geojson";
+  version = "2.3.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06ihcb8839zzgk5jcv18kc6nqld4hhj3nk4f3drzcr8n8893v1y8";
+  };
+
+  # test_unicode_properties fails for Python 3
```

Comment:
Could you post the full error.

Reply:
Here you are:
```
======================================================================
ERROR: test_unicode_properties (test_features.FeaturesTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/geojson-2.3.0/tests/test_features.py", line 41, in test_unicode_properties
    obj = geojson.load(file_)
  File "/build/geojson-2.3.0/geojson/codec.py", line 43, in load
    **kwargs)
  File "/nix/store/96wn2gz3mwi71gwcrvpfg39bsymd7gqx-python3-3.6.5/lib/python3.6/json/__init__.py", line 296, in load
    return loads(fp.read(),
  File "/nix/store/96wn2gz3mwi71gwcrvpfg39bsymd7gqx-python3-3.6.5/lib/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 28: ordinal not in range(128)

----------------------------------------------------------------------
```

Reply:
This is a common issue described in the Nixpkgs manual:

> Unicode issues can typically be fixed by including glibcLocales in buildInputs and exporting LC_ALL=en_US.utf-8.

Reply:
I didn't knew about this one. Thanks for your help!